### ISSUE TITLE
Background for POI cards label is inherited from "outdoor" category color

### ISSUE THREAD

**Issue by dtrucs** [GeotrekCE/Geotrek-rando-v3#1055]
There's no explicit color defined for POI (and I'm not sure it's really relevant).
But the point is that the background is inherited from the "outdoor" category. 
https://github.com/GeotrekCE/Geotrek-rando-v3/blob/main/frontend/src/components/pages/search/components/ResultCard/getActivityColor.ts#L31

This is how it could look on a trek page for example (purple on the left and pink on the right)

![image](https://github.com/GeotrekCE/Geotrek-rando-v3/assets/1926041/0b806e23-ae0e-46b8-b91a-d368c3d7fff5)

Would it be better to define POIs color with the primary1 color, as for the POIs on the map?

**Comment by camillemonchicourt** [comment]
Yes it should not use Outdoor color, and should rather the global and generic primary1 color.

**Comment by dtrucs** [comment]
The best solution is to offer a "poi" settings that lets you define a custom color (with a fallback to the primary color if it's not defined), and also apply this color in the map's POI marker.
